addons/godot_vim/key_map.gd

```diff
@@ -41,14 +41,14 @@ static func map() -> Array[KeyRemap]:
 	# Example:
 	return [
 		# In Insert mode, return to Normal mode with "jk"
-		KeyRemap.new([ "j", "k" ])
-			.action("normal", { "backspaces": 1, "offset": 1 })
-			.with_context(Mode.INSERT),
+		# KeyRemap.new([ "j", "k" ])
+			# .action("normal", { "backspaces": 1, "offset": 0 })
+			# .with_context(Mode.INSERT),
```

Comment:
I think here we are removing behavior. 
Why comment?😅

Reply:
Hi
Those are in the configuration function (seen as how it's in `static func map()`) and not part of the plugin itself, so the thought was to let the user specify it themselves if they want those keybinds.
I figured I'd leave the inside of the returned array commented as an example. I just forgot to comment those 2 ones before 😅